app/src/main/res/values-es/strings.xml

```diff
@@ -352,12 +356,12 @@
     <string name="feedbackIsImportantToUs">Tu opinión anónima nos importa.</string>
 
     <!-- Webview Recovery -->
-    <string name="crashedWebViewErrorMessage">No se pudo mostrar la página web.</string>
-    <string name="crashedWebViewErrorAction">Volver a cargar</string>
+    <string name="crashedWebViewErrorMessage">"No se pudo mostrar la página web."</string>
+    <string name="crashedWebViewErrorAction">"Volver a cargar"</string>
```

Comment:
Noticed these two strings have `"..."`, which is kind of weird. It seems the issue comes from the English strings which were translated with that character.

Reply:
Yeah, some English strings had `"` and they were missing in the translations. Wether they should have quotes in English or not I have no idea 😅

Reply:
This is the error we show in the snackbar when webview crashes, they don't need quotes. Also `crashedWebViewErrorAction` is the button text. We not related to this PR we can fix it in another one later.